core/targets.py

```
@@ -1,19 +1,29 @@
 and physical activity guidelines.
+
+RU: Sistema rascheta tselevykh znacheniy nutrientov na osnove rekomendatsiy VOZ.
+EN: WHO-based nutrition targets calculation system.
 """
 
 from __future__ import annotations
 
 from dataclasses import dataclass, field
 from typing import Any, Dict, List, Literal, Optional, Set
 
+from core.metabolism import (
```

Flake8 (7.3.0)

[error] 'core.metabolism.adjust_calories_for_goal' imported but unused

(F401)